src/core/Akka.API.Tests/CoreAPISpec.ApproveClusterSharding.approved.txt

```diff
@@ -156,70 +179,42 @@ namespace Akka.Cluster.Sharding
         public Passivate(object stopMessage) { }
         public object StopMessage { get; }
     }
+    public enum RememberEntitiesStore
+    {
+        DData = 0,
+        Eventsourced = 1,
+        Custom = 2,
+    }
     public class static ShardAllocationStrategy
     {
         public static Akka.Cluster.Sharding.IShardAllocationStrategy LeastShardAllocationStrategy(int absoluteLimit, double relativeLimit) { }
     }
-    public sealed class ShardInitialized : System.IEquatable<Akka.Cluster.Sharding.ShardInitialized>
+    public sealed class ShardRegion : Akka.Actor.ActorBase, Akka.Actor.IWithTimers
```

Comment:
Is there a reason why we need to change this class to sealed?

Reply:
There is no reason not to. Even if you have your own implementation of `ShardRegion`, you cannot make sharding to use it